tests/ocp/sriov/internal/ocpsriovconfig/config.go

```diff
@@ -88,3 +91,64 @@ func readEnv(sriovOcpConfig *SriovOcpConfig) error {
 
 	return nil
 }
+
+// GetTimestamp returns the current timestamp for report generation.
+// This should be called once at suite start and stored for consistent naming across all reports.
+func GetTimestamp() string {
+	return time.Now().Format("20060102_150405")
+}
+
+// SetReportTimestamp sets the report timestamp to be used for all report files.
+// This ensures consistent timestamps across JUnit and test run reports.
+func (cfg *SriovOcpConfig) SetReportTimestamp(timestamp string) {
+	cfg.reportTimestamp = timestamp
+}
+
+// GetJunitReportPath returns full path to the junit report file with timestamp.
+// Format: {reportsDir}/sriov_suite_test_{timestamp}_junit.xml
+// Uses the timestamp set via SetReportTimestamp() if available, otherwise generates a new one.
+func (cfg *SriovOcpConfig) GetJunitReportPath(file string) string {
+	// Validate reports directory path
+	reportsDir := cfg.ReportsDirAbsPath
+	if reportsDir == "" {
+		// Fall back to current directory if reports path not configured
+		reportsDir = "."
+	}
+
+	reportFileName := filepath.Base(file)
+
+	ext := filepath.Ext(reportFileName)
+	if ext != "" {
+		reportFileName = reportFileName[:len(reportFileName)-len(ext)]
+	}
+
+	// Use stored timestamp if available, otherwise generate new one
+	timestamp := cfg.reportTimestamp
+	if timestamp == "" {
+		timestamp = time.Now().Format("20060102_150405")
+	}
+
+	return fmt.Sprintf("%s_%s_junit.xml", filepath.Join(reportsDir, reportFileName), timestamp)
+}
+
+// GetReportPath returns full path to the reportxml file with timestamp.
+// Format: {reportsDir}/report_{timestamp}_testrun.xml
+// Uses the timestamp set via SetReportTimestamp() if available, otherwise generates a new one.
+func (cfg *SriovOcpConfig) GetReportPath() string {
+	if !cfg.EnableReport {
+		return ""
+	}
+
+	reportsDir := cfg.ReportsDirAbsPath
+	if reportsDir == "" {
+		reportsDir = "."
+	}
+
+	// Use stored timestamp if available, otherwise generate new one
+	timestamp := cfg.reportTimestamp
+	if timestamp == "" {
+		timestamp = time.Now().Format("20060102_150405")
+	}
+
+	return fmt.Sprintf("%s_%s_testrun.xml", filepath.Join(reportsDir, "report"), timestamp)
+}
```

Comment:
I'm not sure that the config file is the right place for these func. The conf file mostly used for the vars and env vars.